integrations/elasticsearch/elasticsearch-events/sensu-integration.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   class: supported
```

Comment:
Should this be `enterprise` instead? sensu/sensu-elasticsearch-handler is listed as an enterprise asset in Bonsai.